src/main/java/org/launchcode/liftoffproject/models/ChildUser.java

```diff
@@ -14,4 +14,12 @@ public ChildUser() {}
     public ChildUser(String username, String password) {
         super(username, password);
     }
+
+    public ChildUser getChildUser() {
+        return null;
+    }
```

Comment:
I don't think we need a getter for the ChildUser, but we should probably add a setter for Child